service/service_construction.go

```diff
@@ -113,9 +120,10 @@ func (s ConstructionService) ConstructionMetadata(
 	}
 
 	metadata := &metadata{
-		Nonce:    nonce,
-		GasPrice: gasPrice,
-		GasLimit: gasLimit,
+		Nonce:          nonce,
+		GasPrice:       gasPrice,
+		GasLimit:       gasLimit,
+		UnwrapBridgeTx: input.UnwrapBridgeTx,
```

Comment:
We should really try to avoid specifying the tx type by providing a metadata arg. This conflicts with some of the Rosetta principles.